alerts/tests.py

```diff
@@ -0,0 +1,30 @@
+from django.core import mail
+from django.test import TestCase
+
+from users.models import User
+from series.models import APIShow
+from alerts.notifications import EmailNotifier
+
+
+class EmailTest(TestCase):
+    """Email notification testing."""
```

Comment:
Since I believe this was heavily inspired by the docs, would you mind adding a reference to it (URL) as a comment?